src/Gateways/GpApiGateway.php

```diff
@@ -195,6 +195,7 @@ public function get_backend_gateway_options() {
 			'methodNotificationUrl'    => WC()->api_request_url( 'globalpayments_threedsecure_methodnotification' ),
 			'challengeNotificationUrl' => WC()->api_request_url( 'globalpayments_threedsecure_challengenotification' ),
 			'merchantContactUrl'       => $this->merchant_contact_url,
+			'txn_descriptor'           => $this->txn_descriptor,
```

Comment:
The dynamic descriptor should be set similar to refund_amount/refund_reason with set_request_data(). This data is used for gateway configuration.

Reply:
Hello @apetrovici  , @ahumulescu 

Can we have a short call tomorrow , just for understanding why my solution is wrong ? I tested and seems to work.

Also, i don't see why the txn_descriptor setted on the gateway config is similar to the refund_amount. 

I added on the AuthorizationRequest.php, the field.. 

Thanks.